tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -158,6 +158,16 @@ Verify "Notebook Images Are Building" Is Not Shown When No Images Are Building
     Wait Until All Builds Are Complete  namespace=redhat-ods-applications
     RHODS Notification Drawer Should Not Contain  message=Notebooks images are building
 
+Verify Switcher to Masterhead
+    [Tags]    ODS-771
+    [Documentation]    Checks the link in switcher and also check the link of OCM in staging
+    Go To RHODS Dashboard
+    Click Button    //button[@class="pf-c-app-launcher__toggle"]
+    ${cluster_id} =    Get Cluster ID
+    ${list_of_links} =    Get Links From Switcher
```

Comment:
could you move this logic to `Check Application Switcher Links To Openshift Console` and `Check Application Switcher Links To Openshift Cluster Manager`?